CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.x.x - Unreleased
+
+### Added
+* Adds preview support for debugging .NET Core web applications running in Linux Docker containers.
```

Comment:
Is it just web apps or also console apps

Reply:
Web apps are the initial target (and the only thing I'm testing at the moment).  Console apps *should* likely be accommodated with perhaps minor tweaks.